packages/auto-chart/src/AdviceList/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import { prefixCls, getThumbnailURL, customChartType } from '../utils';
+
+interface AdviceListProps {
+  advices: any[];
+  currentIndex: number;
+  containerHover: boolean;
```

Comment:
```suggestion
  isActive: boolean;
```

如果是 bool 类型可以重新考虑一下 props 命名，类似 isActive 之类的可以让人一看就知道。containerHover 感觉是个动作而非 bool

Reply:
名称已修改